pulser/sequence.py

```diff
@@ -1081,29 +1081,49 @@ def abstract_repr(self) -> dict[str, Any]:
             ``serialize``
         """
 
-        res = {"register": [], "channels": [], "operations": []}
+        mapping = {
+            "CompositeWaveform": "composite",
+            "CustomWaveform": "custom",
+            "ConstantWaveform": "constant",
+            "RampWaveform": "ramp",
+            "BlackmanWaveform": "blackman",
+            "InterpolatedWaveform": "interpolated",
+            "KaiserWavewform": "kaiser",
+        }
```

Comment:
Actually, I have this being taken care of in a more general way in some local commits. It's still WIP, but I'll push them so that you can see how it's looking. Beware you will get some merge conflicts because I changed this file.

Reply:
This one is clearly "quick and dirty" and I don't intend to keep it for the final version. It was a minimal change so that I can generate JSON for the new schema.